build.xml

```diff
@@ -65,17 +67,36 @@
         </exec>
     </target>
 
-    <target name="phpcpd-ci" description="Find duplicate code using PHPCPD">
+    <target name="phpcpd" description="Find duplicate code using PHPCPD">
         <exec executable="vendor/bin/phpcpd" failonerror="false">
             <arg path="src"/>
         </exec>
     </target>
 
-    <target name="phpunit-ci" description="Run unit tests with PHPUnit">
+    <target name="alltests" description="Run the unit and integration tests" depends="unittests,integrationtests">
```

Comment:
Nit: I'd rather see this and other relevant targets separated by a dash, e.g. `all-tests`, `integration-tests`, `unit-tests`...